### ISSUE TITLE
Getting Started guide

### ISSUE THREAD

**Issue by thany** [eclipse/mosquitto#2993]
Can we please have a getting started guide? I feel the documentation is missing this. There is a page that lists the commands you can use to install, and after that you're basically thrown head-first into detailed reference docs, which are not useful for first-time setup.

Unless installing mosquitto is literally all that's required (which I seriously doubt because it refuses to start on a vanilla docker installation), there needs to be some kind of step-by-step guide on how to setup mosquitto after installing it, and before diving into the all the CLI commands, API details, and protocol implementations and whatnot.

**Comment by sukesh-ak** [comment]
@thany 
You can refer my write-up here to get started
https://github.com/sukesh-ak/setup-mosquitto-with-docker

**Comment by thany** [comment]
I don't think this addresses the chicken&egg problem of getting that `passwd` file - the docker needs to be running to create one, but it requires one to boot up in the first place. On a Windows host, or any other host that doesn't have an easily accessible bash terminal, or one that generates passwd files that are incompatible, this becomes a chicken&egg problem.

Other than that, it's a great guide. What will it take for the maintainers to implement a similarly easy to follow guide here?

**Comment by sukesh-ak** [comment]
> I don't think this addresses the chicken&egg problem of getting that `passwd` file - the docker needs to be running to create one, but it requires one to boot up in the first place. On a Windows host, or any other host that doesn't have an easily accessible bash terminal, or one that generates passwd files that are incompatible, this becomes a chicken&egg problem.
> 
> Other than that, it's a great guide. What will it take for the maintainers to implement a similarly easy to follow guide here?

Maybe you haven't read the readme completely. 
You don't need bash terminal since the `sh` is from the inside of the container which is based on linux.
If you tried and got error, you can open an issue on my repo. I doubt you will get any such help from here though.

The write-up mentioned above was made for self-reference and then realized lot more people are looking for the same.
Good luck 😊
